web-app/src/components/BookingCalendar/index.jsx

```diff
@@ -10,7 +10,13 @@ moment.locale('en-gb');
 Calendar.momentLocalizer(moment);
 
 export const BookingCalendar = props => {
-  const { onSelectSlot, onSelectEvent, events } = props;
+  const { onSelectSlot, onSelectEvent, events, onNavigate } = props;
+
+  const updateOnNavigate = date => {
```

Comment:
Yea you're right. I think I made that to avoid this:
`onNavigate={date => onNavigate(new moment(date))}` which is kinda ugly.

But I've updated it now to transform the date into a moment further down the line instead.